### DISCUSSION TITLE
Resume in @next is great!

### DISCUSSION THREAD

**Discussion by lucacasonato** [tw-in-js/twind#353] [Kind Words]
Hey @sastan. Thanks for your work on this great project. It has been such a pleasure to use in Fresh. We use it on pretty much all of Deno's web properties now.

It's great to see that the @next version can properly resume from SSR sessions. I actually implemented a poor mans resume for twind@0.16.17 for our internal use a while ago that is _very_ hacky but works, and was very excited to see that the @next version supports this out of the box now without requiring any special setup. It's nice to see that we had similar ideas with using CSS comments to achieve this. Thanks for this :)

Here is the cheaty resume code I came up with:
- Server: https://github.com/denoland/dotland/blob/search_box/options.ts#L17-L24
- Client: https://github.com/denoland/dotland/blob/search_box/util/twind.ts#L76-L90

**Comment by renhiyama** [comment]
@lucacasonato thanks for sharing it! It helps out a lot!

**Comment by sastan** [comment]
Thank you @lucacasonato... That is how I would have implemented it as well. Luckily in twind v1 we can do a little better and of cause I/Twind can access some internals.
